api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -4429,3 +4440,106 @@ message LoginRuleDelete {
     (gogoproto.jsontag) = ""
   ];
 }
+
+// SAMLIdPAuth is emitted when a user has attempted to authorize against the SAML IdP.
+message SAMLIdPAuth {
```

Comment:
I'm not an expert on this part of the codebase so I'm merely comparing this to existing messages in this file.

With that in mind, do we have a strong reason to stray from the pattern set out by `AuthAttempt` where a message like this has an "attempt" suffix?

Reply:
Nope, that sounds reasonable to me.